### ISSUE TITLE
build tdlib problem

### ISSUE THREAD

**Issue by CoCoMNT** [tdlib/td#2640]
hi sir
I encountered a problem while building tdlib
LINK : fatal error LNK1181: 无法打开输入文件“ZLIB_LIBRARY-NOTFOUND.lib” [F:\td\build\td\generate\generate_json.vcxproj]

无法打开输入文件=Unable to open input file

**Comment by levlam** [comment]
Did you follow https://tdlib.github.io/td/build.html?

**Comment by CoCoMNT** [comment]
yes  im follow https://tdlib.github.io/td/build.html  other->windows->PowerShell ->32

**Comment by levlam** [comment]
Then, `zlib` should have been installed by the command `/vcpkg.exe install gperf:x86-windows openssl:x86-windows zlib:x86-windows`. You need to check output of the command and whether it succeeded. You can rerun the command if appropriate.
